firefox_puppeteer/ui/toolbars.py

```diff
@@ -137,6 +145,14 @@ def get_contextmenu_entry(self, action):
         return found[0] if len(found) else None
 
     @property
+    def go_button(self):
+        """Provides access to the DOM element go button.
+
+        :returns: Reference to the go button.
+        """
+        return self.marionette.find_element(By.ID, 'go_button')
```

Comment:
I'm not sure what this ID refers to. I think you accidentally put in the property name of the `go_button` you added to the NavBar class. This wont work because the property is not part of the DOM.